src/Container.php

```diff
@@ -397,7 +397,7 @@ protected function getRaw($key)
 
 		$aliasKey = $this->resolveAlias($key);
 
-		if ($aliasKey != $key && isset($this->dataStore[$aliasKey]))
+		if ($aliasKey !== $key && isset($this->dataStore[$aliasKey]))
```

Comment:
I am a bit confused. I wasn't aware of type casting forcing case sensitivity.
See:
```
var_dump('URL' === 'url');
var_dump('URL' == 'url');
var_dump('Url' === 'url');
var_dump('Url' == 'url');
var_dump('url' === 'url');
var_dump('url' == 'url');
```

outputs:
`
bool(false)

bool(false)

bool(false)

bool(false)

bool(true)

bool(true)
`
Or have I misunderstood ?